docusaurus.config.ts

```diff
@@ -67,6 +71,9 @@ const config = (async (): Promise<Config> => {
       tagline_1: 'Build applications in any language.',
       tagline_2: 'Deploy them anywhere.',
     },
+    future: {
+      experimental_faster: true,
```

Comment:
So it really is as easy as setting `faster` to `true` to make the builds quicker? Nice

Reply:
Yeah it switches docusaurus build to use rspack instead of webpack which is a rust implementation of the webpack API